v2/stack_analyses.yaml

```diff
@@ -0,0 +1,469 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/StackAnalysesReport'
+        401:
+          description: Request unauthorized
+          content: {}
+        404:
+          description: Data not found
+          content: {}
+        408:
+          description: Request has timed out on server
+          content: {}
+components:
+  schemas:
+    VulnerabilityForRegisteredUser:
+      title: VulnerabilityForRegisteredUser
+      allOf:
+        - $ref: '#/components/schemas/ExtendedVulnerabilityFields'
+        - oneOf:
+          - $ref: '#/components/schemas/PublicVulnerability'
+          - $ref: '#/components/schemas/PrivateVulnerability'
+      description: Vulnerability details for registered users
+    VulnerabilityForFreeTier:
+      title: VulnerabilityForFreeTier
+      allOf:
+        - $ref: '#/components/schemas/PublicVulnerability'
+      description: Vulnerability details for unregistered users
+    ExtendedVulnerabilityFields:
+      title: ExtendedVulnerabilityFields
+      type: object
+      properties:
+        malicious:
+          type: boolean
+          default: false
+        patch_exists:
+          type: boolean
+        fixable:
+          type: boolean
+        exploit:
+          type: string
+        description:
+          type: string
+        fixed_in:
+          type: string
+        references:
+          type: array
+          items:
+            type: object
+            properties:
+              title:
+                type: string
+              url:
+                type: string
+      description: Extended Vulnerability details, available only to registered users
+    PublicVulnerability:
+      title: PublicVulnerability
+      description: Publicaly known Vulnerability details with associated CVES
+      type: object
+      properties:
+        cve_ids:
+          type: array
+          items:
+            type: string
+        cvss:
+          type: number
+          format: float
+        cvss_v3:
+          type: string
+        cwes:
+          type: array
+          items:
+            type: string
+        id:
+          type: string
+        severity:
+          type: string
+          enum:
+          - low
+          - medium
+          - high
+        title:
+          type: string
+        url:
+          type: string
+    PrivateVulnerability:
+      title: PrivateVulnerability
+      properties:
+        id:
+          type: string
+        severity:
+          type: string
+          enum:
+          - low
+          - medium
+          - high
+        title:
+          type: string
+        url:
+          type: string
+      description: Private vulnerability details
+    Package:
+      title: Package
+      type: object
+      properties:
+        name:
+          type: string
+        version:
+          type: string
+      description: Package details
+    PackageWithTransitive:
```

Comment:
Please check for renaming with plurals

Reply:
It holds single vulnerability details, it is not array of vulnerabilities.

Reply:
Fixed it.